apps/fhir/bluebutton/views/generic.py

```diff
@@ -97,8 +100,11 @@ def get(self, request, resource_type, *args, **kwargs):
     def fetch_data(self, request, resource_type, *args, **kwargs):
         resource_router = get_resourcerouter(request.crosswalk)
         # BB2-291 v2 switch enforced here, entry of all fhir resources queries
+        # interim flag, to be removed after v2 GA
```

Comment:
This comment should probably include `TODO` or `FIXME` since it is introducing tech debt that we will have to clean up later (per the PR template requirements).